pymammotion/mammotion/devices/mammotion.py

```diff
@@ -816,6 +823,7 @@ async def _execute_command_locked(self, key: str, command: bytes) -> bytes:
         except asyncio.TimeoutError:
             timeout_expired = True
             notify_msg = b''
+            self._notify_future.set_result(notify_msg)
```

Comment:
```suggestion
```

Reply:
This will throw an error because the future is cancelled, also kind of pointless too.